projects/mtail/project.yaml

```diff
@@ -0,0 +1,2 @@
+homepage: https://github.com/google/mtail
+primary_contact: jaq@spacepants.org
```

Comment:
you need to specify `fuzzing_engine` and `sanitizer` here as per https://google.github.io/oss-fuzz/getting-started/new-project-guide/go-lang/#projectyaml

sorry this is not a perfect UX, but at least it's documented :) We'll try to have it auto-derived for Go projects some time later.

Reply:
Done in f0ab665